src/test/java/com/powsybl/openloadflow/ac/AreaInterchangeControlTest.java

```diff
@@ -56,49 +58,88 @@ void twoAreasWithUnpairedDanglingLine() {
         double interchangeTarget1 = -60; // area a1 has a boundary that is an unpaired dangling line with P0 = 20MW
         double interchangeTarget2 = 40;
         runLfTwoAreas(network, interchangeTarget1, interchangeTarget2, -10, 3);
+        parameters.setDc(true);
+        runLfTwoAreas(network, interchangeTarget1, interchangeTarget2, -10, 0);
     }
 
     @Test
     void twoAreasWithTieLineTest() {
         Network network = MultiAreaNetworkFactory.createTwoAreasWithTieLine();
         runLfTwoAreas(network, -40, 40, -30, 2);
+        parameters.setDc(true);
+        runLfTwoAreas(network, -40, 40, -30, 0);
     }
 
     @Test
     void twoAreasWithUnconsideredTlTest() {
         Network network = MultiAreaNetworkFactory.createTwoAreasWithUnconsideredTieLine();
         int expectedIterationCount = 3;
         runLfTwoAreas(network, -40, 40, -35, expectedIterationCount);
+        parameters.setDc(true);
+        runLfTwoAreas(network, -40, 40, -35, 0);
     }
 
     @Test
-    void remainingMismatchLeaveOneSlackBus() {
+    void acRemainingMismatchLeaveOnSlackBus() {
+        var mainComponentResult = remainingMismatchLeaveOnSlackBus();
+        assertEquals(-10, mainComponentResult.getDistributedActivePower(), 1e-3);
```

Comment:
@vmouradian: why don't we have a similar assertion on DC unit tests?

Reply:
I forgot to update those tests indeed. They are factorized and made identical for AC and DC in [d3da3ee](https://github.com/powsybl/powsybl-open-loadflow/pull/1123/commits/d3da3ee22652e2dfd91b79fb7012617f3e825bb3)